eslint.config.mjs

```diff
@@ -9,7 +9,10 @@ import jsoncEslintParser from 'jsonc-eslint-parser'
 import eslintPluginJsonc from 'eslint-plugin-jsonc'
 import eslintPluginYml from 'eslint-plugin-yml'
 import yamlEslintParser from 'yaml-eslint-parser'
-import neostandard from 'neostandard'
+// Faster than importing the default import,
+// because the default import imports a lot of other dependencies
+// for the `resolveIgnoresFromGitignore` function that we don't use
+import { neostandard } from 'neostandard/lib/main.js'
```

Comment:
Extension required?
```suggestion
import { neostandard } from 'neostandard/lib/main'
```

Reply:
It seemed to work without it for me, not sure why though as technically the way I wrote it is invalid for ES Module imports, maybe ESLint is doing something weird with the config file (I know they use [jiti](https://github.com/unjs/jiti) on the config file to support TypeScript, maybe that tolerates extensionless imports).

I've added your change now.